### ISSUE TITLE
Poor contrast of standard colors (Non WCAG-compliant)

### ISSUE THREAD

**Issue by hjnilsson** [twbs/bootstrap#20255]
The info, success and warning colors all fail WCAG contrast guidelines. The colors in action on buttons can be seen here: http://v4-alpha.getbootstrap.com/components/buttons/

* Info color ([#83bddc](http://webaim.org/resources/contrastchecker/?fcolor=83bddc&bcolor=ffffff)) fails against white with an extremely poor contrast of 2.
* Warning color ([#dfb057](http://webaim.org/resources/contrastchecker/?fcolor=dfb057&bcolor=ffffff)) also fails with a contrast ratio of 2.
* Success color ([#7fb561](http://webaim.org/resources/contrastchecker/?fcolor= 7fb561&bcolor=ffffff)) fails with a contrast ratio of 2.4.
* Danger color ([#bf5e52](http://webaim.org/resources/contrastchecker/?fcolor= bf5e52&bcolor=ffffff)) passes, with a contrast of 4.4. But this is on the limit (4.5) so pretty much okay.
* Primary color passes.

I suggest improving the colors so they fullfil WCAG guidelines, so that websites built on bootstrap can be used by standard can be used people with vision disabilities (and poor screens). To fulfill this goal they would need to be changed as follows:

*To pass for normal text (<18px):*
Warning: [#a06a23](http://webaim.org/resources/contrastchecker/?fcolor=a06a23&bcolor=ffffff)
Info: [#2b739a](http://webaim.org/resources/contrastchecker/?fcolor=2b739a&bcolor=ffffff)
Success: [#547f3c](http://webaim.org/resources/contrastchecker/?fcolor=547f3c&bcolor=ffffff)

*If we limit to large text (≥18px):*
Warning: [#b78323](http://webaim.org/resources/contrastchecker/?fcolor=b78323&bcolor=ffffff)
Info: [#3b97c8](http://webaim.org/resources/contrastchecker/?fcolor=3b97c8&bcolor=ffffff)
Success: [#619446](http://webaim.org/resources/contrastchecker/?fcolor=619446&bcolor=ffffff)

Related issue regarding validation colors: https://github.com/twbs/bootstrap/issues/19004

**Comment by Florian-R** [comment]
You might want to take a look at #19693

**Comment by hjnilsson** [comment]
#19693 indeed duplicates this. Anticipating it being merged.
